__tests__/client.comprehensive.test.ts

```diff
@@ -324,6 +348,10 @@ describe("MultiServerMCPClient", () => {
         },
       });
 
+      const conf = client.config;
+      expect(conf.additionalToolNamePrefix).toBe("mcp");
+      expect(conf.prefixToolNameWithServerName).toBe(true);
```

Comment:
Added this during test troubleshooting because the defaults reverted due to me accidentally setting `default(...).optional()` in the zod schema updates (needs to be `.optional().default(...)` else the defaults aren't applied during parsing).